src/app/landing/index.tsx

```diff
@@ -0,0 +1,45 @@
+import { Image } from '@chakra-ui/image';
+import { Flex, Grid, GridItem, Heading, Text } from '@chakra-ui/layout';
+import { useMediaQuery } from '@chakra-ui/media-query';
+
+export default function Landing() {
+  const [isLargerThan1100] = useMediaQuery('(min-width: 1100px)');
+  console.log(isLargerThan1100);
+
+  return (
+    <Grid
+      templateColumns={isLargerThan1100 ? 'repeat(2, 1fr)' : 'repeat(1, 1fr)'}
```

Comment:
Is there anything stopping using the `xl` breakpoint value to accomplish this? 
`templateColumns={{xl: 'repeat(2, 1fr)', default: 'repeat(1, 1fr)'}}}`? (or something like that)

Reply:
only problem is using breakpoints makes it inconsistent with https://github.com/VikeLabs/clockwork/pull/118/files#diff-5556e38476f1d1d808a8977a1b10585e696de85182714f6f9fa62108f1cd1565R31, so the computer image will go under the text for a bit, and then disappear. this way does both at the exact same width

Reply:
both as in going to 1 column and getting rid of image